.github/workflows/test-smoke.yml

```diff
@@ -128,7 +128,7 @@ jobs:
             name: kyma-cli-provisioned-wildcard
           subjects:
             - kind: ServiceAccount
-              name: lifecycle-manager-controller-manager
+              name: lifecycle-manager-controller
```

Comment:
this should be `lifecycle-controller-manager`?

Reply:
True, will fix 👍🏻

Reply:
fixed here https://github.com/kyma-project/lifecycle-manager/pull/1721/commits/76758f0a88630080dc412f2bd878c0e1f71ce459